source/whereAny.js

```diff
@@ -0,0 +1,47 @@
+import _curry2 from './internal/_curry2';
+import _has from './internal/_has';
+
+
+/**
+ * Takes a spec object and a test object;
+ * Each of the spec's own properties must be a predicate function.
```

Comment:
Because the prior phrase ends in a semicolon, `Each` should be `each`. We can also move it up onto the same line, breaking in the typical spot.

Alternately, we could use a period, or rewrite the sentence, but I think that changing E to e is easiest.

Reply:
- [ ] fix documentation

Reply:
@Bradcomp :) we should be good to go!! please, take a look at my latest changes :)